src/orion/algo/mofa/sampler.py

```diff
@@ -0,0 +1,110 @@
+"""
+MOFA Sampler module
+"""
+
+# from skopt.sampler import Lhs
+# from skopt.space import Space as SkSpace
+
```

Comment:
```suggestion
```

Reply:
Addressed in https://github.com/Epistimio/orion/pull/864/commits/b3f58efb4dc69ffc97b85c11b959110c69933d4e